pom.xml

```diff
@@ -120,6 +121,18 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>3.2.0</version>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <Export-Package>software.amazon.ion.*</Export-Package>
```

Comment:
Everything under `software.amazon.ion.impl` is non-API and should _not_ be exported.

Further, any class named `Private*` is also non-API and should not be exported; is that possible?